### PR TITLE
Enable access to SLD layer style through Layman REST API

### LINKED ISSUE #96: Enable access to SLD layer style through Layman REST API

### PR DESCRIPTION
**Pull request by index-git**
Close #96

### CODE REVIEW

File: src/layman/layer/rest_layer_test.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+from layman.util import url_for
+
+
+client = flask_client.client
+
+
+def test_sld_value(client):
+    username = 'test_layer_sld_user'
+    layername = 'test_layer_sld_layer'
+    style_url = f'http://{settings.LAYMAN_SERVER_NAME}/rest/{username}/layers/{layername}/style'
```
Comment (jirik) [inline src/layman/layer/rest_layer_test.py:16]: Should be settings.LAYMAN_PROXY_SERVER_NAME, not LAYMAN_SERVER_NAME.

**Review by jirik — CHANGES_REQUESTED** [review]

File: src/layman/layer/geoserver/sld.py [hunk 2]
```diff
@@ -54,7 +54,18 @@ def delete_layer(username, layername):
 
 
 def get_layer_info(username, layername):
-    return {}
+    r = get_style_response(username, layername, headers_xml, settings.LAYMAN_GS_AUTH)
+    if r.status_code == 200:
+        url = f'http://{settings.LAYMAN_SERVER_NAME}/rest/{username}/layers/{layername}/style'
```
[the remark this answers is not in the document]
Comment (jirik) [reply src/layman/layer/geoserver/sld.py:59]: Actually, you need to care also about HTTP scheme (http vs https). I recommend you to generate this URL using same approach as in src/layman/layer/filesystem/thumbnail.py get_layer_info using url_for from layman.util. Something like
```
from layman.util import url_for
...
def get_layer_info(username, layername):
    r_url = url_for('rest_layer_style.get', username=username, layername=layername)
   ...
```

[the remark this answers is not in the document]
Comment (index-git) [reply src/layman/layer/geoserver/sld.py]: Good point.

File: src/layman/layer/rest_layer_test.py (hunk 1 above)
Comment (index-git) [reply src/layman/layer/rest_layer_test.py]: Actually, url_for :)

**Review by jirik — APPROVED** [review]

**Pull request merged**